gulp/tasks/default.js

```diff
@@ -2,13 +2,13 @@ var gulp = require('gulp');
 var config = require('../config');
 
 gulp.task('watch', function() {
-    gulp.watch(config.browserify.watch,   ['browserify-app']);
+    gulp.watch(config.browserify.watch,   ['browserify']);
     gulp.watch(config.css.watch,          ['css']);
     gulp.watch(config.resources.watch,    ['resources']);
     gulp.watch(config.examples.watch,     ['examples']);
     gulp.watch(config.examplesJS.watch,   ['examplesJS']);
 });
 
-gulp.task('build-all', ['browserify-app', 'browserify-minify-app', 'vendor', 'css', 'resources', 'examples', 'examplesJS']);
+gulp.task('build-all', ['browserify', 'vendor', 'css', 'resources', 'examples', 'examplesJS']);
```

Comment:
Good catch. My goal was to simplify and make it more consistent with the way the CSS is handled (and the way browserify is handled in CFM), but I failed to add 'browserify-minify-app' to the 'browserify' task in browserify-task.js. I'll make that change as well as putting back the 'browserify-app' in watch so we don't minify in that case.